.github/workflows/integration.yml

```diff
@@ -63,7 +63,7 @@ jobs:
 
       # Test run video was always captured, so this action uses "always()" condition
       - name: upload Cypress test run video
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v3
```

Comment:
There's also another usage of `actions/upload-artifact@v2` on line 58

Reply:
ooh, missed that one, thanks!  I've updated that.